### PR TITLE
Notebook server validation

### PR DESCRIPTION
**Pull request by saffaalvi**
Resolves https://github.com/StatCan/jupyter-apis/issues/7

### CODE REVIEW

**Review by zachomedia — APPROVED** [review]
Assuming this has been tested, this LGTM!

**Comment by brendangadd** [comment]
Converting to draft; I want to push something to this before it merges.

**Review by brendangadd — CHANGES_REQUESTED** [review]
Form accepts raw numbers for RAM as valid which, defaulting to Mi, allows values outside the valid range.

**Review by brendangadd — APPROVED** [review]

**Pull request merged**
